frontend/webapp/views/actions/actions_view.py

```diff
@@ -130,6 +130,9 @@ def remove_admin():
     if form.select_field.data is not None:
         user = app.security.datastore.find_user(
             id=form.select_field.data)
+        # Protect admin (user id == 1)
+        if user.id == 1:
+            return redirect('admin/user_management')
```

Comment:
returns without a prompt?

Reply:
Nope. Its for extra security. Manually this action can't be triggered